src/performers.ts

```diff
@@ -32,6 +32,7 @@ poseEmitter.on("pose", (pose: BlazePose.Pose) => {
       name: username,
       isSelf: true,
       connected: true,
+      appearance: settings.apperance, // TODO: double-check if this updatePersonPose is local
```

Comment:
This should be `settings.appearance` (with an `a` after the `e`)

Reply:
fixed!